_test/general.test.php

```diff
@@ -374,18 +384,18 @@ public function test_plugin_setting_exclude_pages()
         $expectedOrdernumber = $pageIds[0];
         $expectedItems = 1;
 
-        $this->assertEquals($expectedTotal, $total, 'Expected total value should match "1" when two of three total pages are excluded in the configuration.');
+        $this->assertEquals($expectedTotal, $total, 'Expected total value should be one when two of three total pages are excluded in the configuration.');
         $this->assertEquals($expectedOrdernumber, $ordernumber, 'Expected ordernumber should match "settingtest1337" when two of three total pages are excluded in the configuration.');
```

Comment:
Leave out the excepted value, it's added automatically by PHPUnit

Reply:
done